include/vctr/Containers/Vector.h

```diff
@@ -174,16 +174,30 @@ class Vector : public VctrBase<ElementType, std::vector<ElementType, Allocator<E
         return { Vctr::begin(), Vctr::end() };
     }
 
+    /** Standard interface function forwarded to std::vector. */
     constexpr void resize (size_t i) { Vctr::storage.resize (i); }
 
+    /** Standard interface function forwarded to std::vector. */
     constexpr void reserve (size_t i) { Vctr::storage.reserve (i); }
 
+    /** Standard interface function forwarded to std::vector. */
+    constexpr void shrink_to_fit() { Vctr::storage.shrink_to_fit(); }
+
+    /** Standard interface function forwarded to std::vector. */
     constexpr void push_back (ElementType&& newElement) { Vctr::storage.push_back (std::move (newElement)); }
 
+    /** Standard interface function forwarded to std::vector. */
     constexpr void push_back (const ElementType& newElement) { Vctr::storage.push_back (newElement); }
 
+    /** Standard interface function forwarded to std::vector. */
     template <class... Args>
     constexpr void emplace_back (Args&&... args) { Vctr::storage.emplace_back (std::forward<Args> (args)...); }
+
+    /** Standard interface function forwarded to std::vector. */
+    constexpr size_t capacity() const noexcept { return Vctr::storage.capacity(); }
+
+    /** Standard interface function forwarded to std::vector. */
+    constexpr Allocator<ElementType> get_allocator() const noexcept { return Vctr::storage.get_allocator(); }
```

Comment:
I'm not sure if I prefer this comment style since it does not communicate directly what the functions are actually doing but requires the user to look up the `std::vector` documentation to find that out. I'd rather prefer e.g.

```
Returns the allocator associated with the Vector.

This is a standard interface function forwarded to std::vector::get_allocator.
```